example/server.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"github.com/opentracing-contrib/go-restful"
+	zipkin "github.com/openzipkin/zipkin-go-opentracing"
+)
+
+func main() {
+	// 1) Create a opentracing.Tracer that sends data to Zipkin
+	collector, _ := zipkin.NewHTTPCollector(fmt.Sprintf("http://127.0.0.1:9411/api/v1/spans"))
+	// set tracer
+	otrestful.Tracer, _ = zipkin.NewTracer(
```

Comment:
You are going to have race conditions, setting global variables directly in this manner. Use an init function to explicitly initialize these, with a lock. Also, please use the GlobalTracer by default (https://godoc.org/github.com/opentracing/opentracing-go#GlobalTracer).